### PR TITLE
Describe our use of the "degree" terminology.

### LINKED ISSUE #124: Use of term "degree"
From #111:

> This use of 'degree' is just plain wrong, although I fear it might be too deeply entrenched for you to consider changing. The standard meaning of 'degree' in the context of graphs is as a synonym of valency (number of incidences of edges to the vertex under consideration). The concept at play here is rather that of radius of the neighbourhood being considered. I suppose for bounded valency graphs you could arrive at the radius being the degree of the polynomial bound on the cardinality of the neighbourhood considered, but even that is a far-fetched choice of terminology; depth (as in 'recursion depth') seems more natural.

> However, the risk of confusion is probably low.

### PR DESCRIPTION
**Pull request by gkellogg**
Fixes #124.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-canon/pull/139.html" title="Last updated on Jul 14, 2023, 3:16 PM UTC (37beed4)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-canon/139/5c8e5e6...37beed4.html" title="Last updated on Jul 14, 2023, 3:16 PM UTC (37beed4)">Diff</a>

### CODE REVIEW

File: spec/index.html [hunk 1]
```diff
@@ -508,6 +508,15 @@ <h3>Overview</h3>
       hexadecimal representation of the result of passing <var>s</var>
       through a cryptographic hash function.
       <a>RDFC-1.0</a> uses the SHA-256 hash algorithm.</p>
+
+    <p class="note">The "degree" terminology is used within this specification
+      as coloquial way of describing
+      the <a href="https://en.wikipedia.org/wiki/Distance_(graph_theory)#Related_concepts">excentricity</a> or
```
Comment (iherman) [inline spec/index.html:514]:
```suggestion
      the <a href="https://en.wikipedia.org/wiki/Distance_(graph_theory)#Related_concepts">eccentricity</a> or
```

(This is the spelling used in that Wikipedia article...)

File: spec/index.html [hunk 2]
```diff
@@ -508,6 +508,15 @@ <h3>Overview</h3>
...
+      <a href="https://en.wikipedia.org/wiki/Distance_(graph_theory)#Related_concepts">radius</a>
+      of any two nodes within a dataset.
+      The concept here is related to "degrees of separation",
```
Comment (iherman) [inline spec/index.html:517]:
```suggestion
      The concept here is also related to "degrees of separation",
```

**Review by iherman — APPROVED** [review]
(Just for the fun: I see that the <a href="https://en.wikipedia.org/wiki/Six_degrees_of_separation">six degrees of separation</a> article refers to Frigyes Karinthy, a very well known Hungarian humorist and writer, as the originator of the concept and, at least for mathematicians,  the [Erdős number](https://en.wikipedia.org/wiki/Erd%C5%91s_number) is the best example for the concept… Conclusion: be aware of Hungarians. It is a dangerous crowd and they are everywhere 😀)

File: spec/index.html [hunk 3]
```diff
@@ -511,10 +511,10 @@ <h3>Overview</h3>
 
     <p class="note">The "degree" terminology is used within this specification
       as coloquial way of describing
-      the <a href="https://en.wikipedia.org/wiki/Distance_(graph_theory)#Related_concepts">excentricity</a> or
+      the <a href="https://en.wikipedia.org/wiki/Distance_(graph_theory)#Related_concepts">eccentricity</a> or
       <a href="https://en.wikipedia.org/wiki/Distance_(graph_theory)#Related_concepts">radius</a>
       of any two nodes within a dataset.
-      The concept here is related to "degrees of separation",
+      The concept here is also related to "degrees of separation",
       for example as in <a href="https://en.wikipedia.org/wiki/Six_degrees_of_separation">six degrees of separation</a>.
```
Comment (TallTed) [inline spec/index.html]:
```suggestion
      This concept is also related to "degrees of separation",
      as in, for example, "<a href="https://en.wikipedia.org/wiki/Six_degrees_of_separation">six degrees of separation</a>".
```

**Review by dlongley — APPROVED** [review]
+1 with @TallTed's suggestions. Thanks!

**Pull request merged**